docs/_layouts/past-results-layout.html

```diff
@@ -54,9 +54,8 @@ <h2>{{page.subheading}} Integration Tests</h2>
 </script>
 
 <style>
-  td:hover,
-  tr:hover {
-    background-color: #ccfff9;
+  td.details table tr:hover td{
+    background-color: var(--hidden-details-data-color) !important;
```

Comment:
why did you use "!important"?

Reply:
> why did you use "!important"?

@tofikwest 
!important is ensuring that the specified styles (background-color and cursor) are applied, even if there are other conflicting styles defined elsewhere.
this was present almost everywhere where we are using variable colors in table.scss, so i used it here.